qstash/workflow/roadmap.mdx

```diff
@@ -6,10 +6,9 @@ title: "Roadmap"
     - Ordered queues
     - Parallelism limits
     - Native LLM calls
-    - Configure retry limit(right now it uses default 3)
-- Expand language support with additional SDKs:
-    - Python
-    - Other languages based on user requests
+    - Configure (step-level) retry limit (currently it's only possible to set retry for all steps at once)
+- Expand language support with additional SDKs: Python and other languages based on user requests.
 - Introduce events to trigger multiple endpoints
-- Implement a Wait/Notify mechanism to pause execution and resume upon an external event
-- Local development environment for testing
\ No newline at end of file
+- Local development environment for testing
+- Endpoint for checking the status & progress of a workflow run
+- Running in Vercel Preview
```

Comment:
wdym by this?

Reply:
you can't run workflow in vercel preview, because it requires auth. An alternative is to use [protection bypass](https://vercel.com/docs/security/deployment-protection/methods-to-bypass-deployment-protection/protection-bypass-automation) somehow, but that doesn't work either because the header can't be forwarded for subsequent requests.